pandas/io/common.py

```diff
@@ -324,9 +325,51 @@ def _get_handle(path, mode, encoding=None, compression=None):
         else:
             f = open(path, mode)
 
+    if memory_map and hasattr(f, 'fileno'):
+        try:
+            f = MMapWrapper(f)
+        except:  # fallback: leave file handler as is
```

Comment:
@gfyoung If you scroll down in `parser.pyx` [a little further](https://github.com/pydata/pandas/blob/master/pandas/parser.pyx#L626), you notice if the pointer is `NULL` we raise `IOError`.

I don't think we should ignore arbitrary errors, certainly not if this is disabled by default. Even then, never use blanket `except:` clauses -- `except Exception:` is much safer.